config/routes.rb

```diff
@@ -101,10 +101,10 @@
     end
   end
 
+  patch 'supervisors/notes', to: 'supervisors/notes#update'
```

Comment:
I'm curious, what let do needing this change? I'm wondering only because the former looked nicer 😄. However, please feel free to disregard this as the rest looks great.

Reply:
> The Supervisor Dashboard didn't work anymore, because the routes to the notepad and the comments were broken (after the namespace makeover in #893).

The dashboard is not reachable. Reproduce: assign your self a supervisor role, click on the link to supervisor dashboard -> 💥

Reply:
I think another solution to this is to change `resources` to `resource`, which may be a bit more rails-y: `resource :notes, only: [:update]`.

Update: as I was thinking about this, I pulled the branch and tried it out. I noticed the error that the regular `resources :notes, only: [:update]` was getting me is that `supervisors_note_path` requires a note ID, which makes sense, since we would be updating an existing record.

Replacing line 30 with this form, makes it work:

```slim
= simple_form_for [:supervisors, Note.notepad(current_user)] do |f|
```

It uses the `note_path`, but namespaced to `supervisors`.

On the other hand, if we really want to make it so no ID is ever passed, because we will always be editing the same Note, I'd suggest using `resource`, since I think it [matches our intention here](http://guides.rubyonrails.org/routing.html#singular-resources). In that case, maybe it should be `resource :note` and not `noteS`.

Reply:
😍